sklearn/datasets/tests/test_openml.py

```diff
@@ -0,0 +1,116 @@
+"""Test the openml loader.
+"""
+import json
+import numpy as np
+import unittest.mock
```

Comment:
This isn't available in Python 2. I wasn't really intending that you use this module... I was just wondering if there was a way to test our response to various HTTP sequences without requiring openml to keep the same data.

Reply:
What would be pretty cool is to have a mocked test in the sklearn unit tests, and a live one in the OpenML-Python unit tests (we have so many live tests, one or two additional tests don't matter anyway). 

I'll explore the options to override the web-based functions without the mock library